content/oci/oci.go

```diff
@@ -52,12 +53,24 @@ type Store struct {
 	//     to manually call SaveIndex() when needed.
 	//   - Default value: true.
 	AutoSaveIndex bool
-	root          string
-	indexPath     string
-	index         *ocispec.Index
-	storage       *Storage
-	tagResolver   *resolver.Memory
-	graph         *graph.Memory
+
+	// AutoGC controls if the OCI store will automatically clean newly produced
+	// dangling nodes during Delete() operation. Dangling nodes are those without
+	// any predecessors, such as blobs whose manifests have been deleted.
+	//   - Default value: true.
+	AutoGC bool
+
+	// AutoRemoveReferrers controls if the OCI store will automatically delete its
+	// referrers when a manifest is deleted.
+	//   - Default value: true.
+	AutoRemoveReferrers bool
```

Comment:
You also want to document that the referrers will be removed even if they are in the `index.json`.

Reply:
Added the documentation.